common/js/src/messaging/message-channel-pool.js

```diff
@@ -16,16 +16,17 @@
 
 /**
  * @fileoverview This file containts the pool that keeps track of all the
- * Closure-style AbstractChannel's used for messaging
- * (PortMessageChannel & SingleMesageBasedChannel).
+ * Closure-style AbstractChannel's used for messaging (PortMessageChannel &
```

Comment:
> This class works regardless of the concrete element types.

It does. I wanted to clarify the statement a bit by adding the concrete class names. Maybe I should prefix the names with _eg._